wix-bookings-v2/wix-bookings-v2/Bookings/guides/Introduction.md

```diff
@@ -23,69 +23,7 @@ With Wix Bookings V2 API you can retrieve existing bookings and manage the booki
-      owner and is displayed in the business calendar.
-      You can't manually set bookings as `PENDING` via an API.
-      Bookings are automatically set as `PENDING` when an eCommerce order related to the booking has been created.
-    + `WAITING_LIST`: The booking is pending on a waiting list.
-    + `DECLINED`: The booking has been declined by the site owner.
-        + You can manually decline a booking with 
-          [Decline Booking()](#declineBooking).
-        + Bookings are automatically declined when an eCommerce order has been 
-          declined or a double booking happened for free bookings.
-    + `CANCELED`: The booking has been canceled by the site owner or the customer.
-        You can cancel bookings with [Cancel Booking()](#cancelBooking).
-+ __Double booked__:
-  A double booking can happen in case of 2 or more simultaneous eCommerce checkout 
-  processes or if the site owner manually accepts bookings that exceed the 
-  service's capacity limit.
-  + When a double booking happens for paying customers, the `doubleBooked` 
-    boolean is set to `true` for the second booking and the owner is notified. 
-    They must then manually resolve the situation.
-  + When a double booking happens for free services, the second 
-    booking's status is set to `DECLINED` while the first booking is retained.
-+ __Service__: Business offering. For example, a fitness studio may offer a 
-  1-hour yoga class, a 45 minute HIIT training session, and a 30 minute 1-on-1 
-  personal training. Learn more about [services](https://support.wix.com/en/article/creating-the-right-booking-service-for-your-business).
-+ __Variant__: Unique variation of the service. Each service variant may have 
-  a different price.
-+ __Availability__: Free time on a schedule that customers can book.
-+ __Schedule__: Entity that holds availability information. A 
-  schedule can 
-  either belong to a resource (for example, a staff member) or a service. 
-+ __Session__: An occupied period of time on a schedule. When 
-  creating a booking (coming soon) 
-  for a class you need to pass the service's relevant existing session as 
-  `slot` in the body of the request.
-+ __Slot__: An available period of time in a schedule that can be booked by a 
-  customer.
-+ __Deposit__: Amount the customer must pay immediately when checking out the 
-  booking.
-+ __Visitor__: Anyone who isn't registered as a member or hasn't logged in to 
-  the Wix site. If a visitor creates a booking they're added as a contact.
-+ __Member__: Someone who is a registered member in the Wix site and has a 
-  login.
-+ __[Wix Pricing Plans](https://support.wix.com/en/article/about-pricing-plans)__: 
-  A pre-paid bundle of services or a membership that provides access to certain
-  services.
-
+For a comprehensive glossary of Wix Bookings terms, see [Terminology](https://www.wix.com/velo/reference/wix-bookings-v2/terminology).
```

Comment:
You are missing the "Before you begin" heading before the line that says "To use the Bookings V2 API..." 
Right now it looks like that line is part of Terminology.

Reply:
I believe "Before you Begin is supposed to be before Terminology.  Please review the intro article guidelines in the cafe.

Reply:
fixed